integration-test/build.gradle.kts

```diff
@@ -29,7 +29,6 @@ dependencies {
   testImplementation(project(":server"))
   testImplementation(project(":server-common"))
   testImplementation(project(":spark-connector")) {
-    exclude("org.apache.iceberg")
```

Comment:
why remove this?

Reply:
some class not found if exclude iceberg dependency.